apps/chat/src/utils/app/conversation.ts

```diff
@@ -93,16 +93,21 @@ export const getNewConversationName = (
   conversation: Conversation,
   message: Message,
 ): string => {
-  const convName = prepareEntityName(conversation.name);
   const content = prepareEntityName(message.content);
+
   if (content.length > 0) {
     return content;
   } else if (message.custom_content?.attachments?.length) {
-    const files = message.custom_content.attachments;
-    return files[0].title;
+    const { title, reference_url } = message.custom_content.attachments[0];
+
+    if (!title && reference_url) {
+      return reference_url;
+    }
+
+    return title;
   }
 
-  return convName;
+  return conversation.name;
```

Comment:
please, revert this change with `prepareEntityName(conversation.name);` and add `prepareEntityName(reference_url)`